lib/locales/en.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
```

Comment:
@ramzieus maybe it's better to put this code into the `tago.rb` file. If we keep it in the `locales/en.rb`, it may easily get into conflict with other files similarly named, from other packaged installed by the user.

It's better to keep this entire small library in a single source file: `tago.rb`